src/Markdown.MAML/Renderer/Markdownv2Renderer.cs

```diff
@@ -534,26 +604,32 @@ private string GetAutoWrappingForMarkdown(string[] lines)
             return string.Join(NewLine, newLines);
         }
 
-        private void AddParagraphs(string body, bool noNewLines = false)
+        private void AddParagraphs(string body, bool noNewLines = false, bool removeWhitespace = false)
         {
             if (string.IsNullOrWhiteSpace(body))
             {
                 return;
             }
 
+            var bodyFormatted = body;
+            if (removeWhitespace)
```

Comment:
it seems that `removeWhitespace` really should be named something like `replacNewLinesBySingleNewLine` , is it?